docker-compose.medea.yml

```diff
@@ -0,0 +1,16 @@
+version: "2"
+
+services:
+  medea:
+    container_name: ${COMPOSE_PROJECT_NAME}-backend
+    image: ${COMPOSE_IMAGE_NAME}:${COMPOSE_IMAGE_VER}
+    ports:
+      - "8080:8080"
+    volumes:
+      - ${MEDEA_SERVER_STATIC_SPECS_PATH}:/specs
+      - ./config.toml:/config.toml
```

Comment:
Also, this `config.toml` should not be used for development. It's only a reference with documentation.

For developing purposes we have `dev/` dir where dev environment confs may be placed naturally.